tests/exportImage.py

```diff
@@ -44,6 +46,14 @@ def test_invalid_export(self):
         result = is_valid_zoom_for_area(resolution, bbox)
         self.assertEqual(result, False)
 
+    # Check that correct overviews are calculated
+    def test_calculate_overviews_correctly(self):
+        fullPath = f'{path.dirname(__file__)}/test.GPKG'
+        Image = gdal.Open(fullPath, 1)
+        overviews = calculate_overviews(Image, 18)
+        # Should have overviews 17, ..., 9
+        self.assertEqual(overviews, [2, 4, 8, 16, 32, 64, 128, 256, 512])
```

Comment:
shouldn't be more?

Reply:
No, not for the size of the original raster